compare_benchmarks/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "compare_benchmarks"
```

Comment:
I was thinking if we should just have a scripts folder with a set of rust scripts. Instead of a cargo project.

https://github.com/fornwall/rust-script